_includes/_blogcontent/2024-02-26-userfeedback-2024.md

```diff
@@ -0,0 +1,68 @@
+The virtual gathering of the [Astrophysics Data System Users Group (ADSUG)](https://ui.adsabs.harvard.edu/about/adsug/) on 16-17 November served as a pivotal moment in our ongoing transition from the Astrophysics Data System (ADS) to the [NASA Science Explorer (SciX)](https://scixplorer.org/scixblog/scix). A crucial precursor to this meeting was a user survey, offering a comprehensive view of the user community's perceptions, expectations, and concerns from the 248 survey respondents. 
```

Comment:
For links to other pages with the about/help/blog pages, we use relative linking so we don't accidentally break links at some point in the future. It can be a little tricky at first (that's partly why I always double check the links) but it looks like this:
* `https://ui.adsabs.harvard.edu/about/adsug/` should be `../about/adsug/`
* `https://scixplorer.org/scixblog/scix` should be `../scixblog/scix`

Reply:
Should we include a line about where we advertised the survey?

Reply:
The page was updated with relative linking and additional information was included about where the survey was advertised.